gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Mar 28 23:53:18 MSK 2017
+#Fri Jun 19 14:34:22 TRT 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.4.1-all.zip
```

Comment:
Would it take long to replace gradle by sbt?

Reply:
I'd prefer to open an issue and handle it next time due to urgency on snowplow/enrich